src/django/oar/settings.py

```diff
@@ -129,6 +130,23 @@
     'PAGE_SIZE': 20,
 }
 
+SWAGGER_SETTINGS = {
+    'SECURITY_DEFINITIONS': {
+        'api_key': {
+            'type': 'apiKey',
+            'in': 'header',
+            'name': 'Authorization',
+        },
+    },
+    'doc_expansion': 'list',
+    'info': {
+        'description': 'Open Apparel Registry API',
+        'license': 'MIT',
+        'licenseUrl': 'https://github.com/open-apparel-registry/open-apparel-registry/blob/develop/LICENSE', # noqa
+        'title': 'Open Apparel Registry API',
+    },
+    'USE_SESSION_AUTH': False,
```

Comment:
I think we may be able to configure the base url somewhere here so I'll try it out tomorrow.

Reply:
This ended up not working. I asked a question here https://github.com/marcgibbons/django-rest-swagger/issues/681 about whether there's an alternate way to fix this, so if someone answers we can circle back to this later.